include/z64actor.h

```diff
@@ -498,6 +498,29 @@ typedef enum {
     /* 0xFF */ NAVI_ENEMY_NONE = 0xFF
 } NaviEnemy;
 
+#define TRANSITION_ACTOR_PARAMS_INDEX_SHIFT 10
+#define TRANSITION_ACTOR_INDEX(actor) ((u16)(actor)->params >> TRANSITION_ACTOR_PARAMS_INDEX_SHIFT)
+
+// EnDoor and DoorKiller share openAnim and playerIsOpening
+// Due to alignment, a substruct cannot be used in the structs of these actors.
+#define DOOR_ACTOR_BASE               \
+    /* 0x0000 */ Actor actor;         \
+    /* 0x014C */ SkelAnime skelAnime; \
+    /* 0x0190 */ u8 openAnim;         \
+    /* 0x0191 */ u8 playerIsOpening;
```

Comment:
Well admittedly this is a dumb solution anyway but it's the less dumb I found, I have the `;` to make it look more like a normal struct but if you think it would be better as `/* 0x0000 */ DOOR_ACTOR_BASE;` then sure?

Reply:
do while 0 instead to force semicolon? provided it doesnt break matching anywhere lol

Reply:
in a struct definition? :morphon:

Reply:
or wait this gets used in headers, not in code blocks. so prob not lol

Reply:
`/* 0x0000 */ DOOR_ACTOR_BASE;` seems slightly more appropriate to me, since this is meant to be used inside a struct similarly to a sub struct